site/examples/richtext.js

```diff
@@ -37,6 +37,7 @@ const RichTextExample = () => {
       <Editable
         renderElement={renderElement}
         renderLeaf={renderLeaf}
+        // See slate-react docs for how to style a custom placeholder.
```

Comment:
I don't think this comment will compile because it's amongst JSX, I think it's okay to remove.

```suggestion
```

Reply:
These comments are valid if you're commenting within a component block and are invalid if you are commenting in the render block of JSX (need the fancier comments there). I'll go and remove to keep consistency with other examples.

https://codesandbox.io/s/awesome-feistel-1xuiz?file=/src/App.js